test/regressions/package.json

```diff
@@ -3,11 +3,11 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "start": "cross-env NODE_ENV=production pnpm build && concurrently --success first --kill-others \"pnpm run mocha\" \"pnpm run server\"",
+    "start": "cross-env NODE_ENV=production pnpm build && concurrently --success first --kill-others \"pnpm run test\" \"pnpm run server\"",
     "build": "cross-env TEST_BUILD=true vite build",
     "dev": "concurrently \"pnpm run build --watch\" \"pnpm run server\"",
-    "mocha": "mocha --config .mocharc.js '**/*.test.{js,ts,tsx}'",
-    "server": "serve . -p 5001"
+    "test": "vitest",
+    "server": "serve . -L -p 5001"
```

Comment:
Why do we need this `-L` flag? Is it a leftover from tests you made or is it mean to be kept?